recipe/build.sh

```diff
@@ -9,7 +9,7 @@ export HDF5_LDFLAGS="-L ${PREFIX}/lib"
 ./configure --prefix=${PREFIX} --with-hdf5=${PREFIX} --with-zlib=${PREFIX}
 
 make
-make check
+#make check
```

Comment:
Can you write a comment here with the failure or a summary of the failure?

Reply:
One of the sample programs is segfaulting, but this seems to have been an issue even with HDF-EOS5 1.15 and HDF5 1.8.18. Specifically, the segfault occurs when running `./he5_pt_readattrs`:
```make[2]: *** [pt_write_test] Segmentation fault (core dumped)```

I will report this to the HDF-EOS5 developers so that we can hopefully un-comment `make check` in `build.sh` in the future.